js/main.js

```diff
@@ -0,0 +1,345 @@
+  linkToLive: 'https://github.com/solorics',
+  btn: 'See Source',
+  side: ['Next Project', 'Previous Project'],
+  icons: ['./assets/icons/Vector.png', './assets/icons/Union.png'],
+  btnForPopup: ['See Live', 'See Source'],
+  nextPrevious: ['./assets/icons/pre.png', './assets/icons/next.png'],
+}, {
+  id: '6',
+  class: 'item10',
+  title: 'Project name goes here',
+  features: ['HTML/CSS', 'Ruby on Rails', 'JavaScript'],
+  image: './assets/images/Rectangle26.svg',
+  img: './assets/images/Union.svg',
+  sideArrows: ['./assets/icons/left.png', './assets/icons/right.png'],
+  sideImage: [
+    './assets/images/Rectangle26.svg',
+    './assets/images/Rectangle26.svg',
+    './assets/images/Rectangle26.svg',
+    './assets/images/Rectangle26.svg',
+  ],
+  description: `Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut
+enim ad minim veniam, quis nostrud exercitation ullamco laboris
+<br /><br />
+nisi Ut aliquip ex ea commodo consequat. Lorem ipsum dolor sit
+amet, consectetur adipiscing elit, sed do eiusmod tempor
+incididunt ut labore et dolore magna aliqua. Ut enim ad minim
+veniam, quis nostrud exercitation ullamco laboris nisi.`,
+  linkToSource: 'https://solorics.github.io/Portfolio-Mobile-FIrst/',
+  linkToLive: 'https://github.com/solorics',
+  btn: 'See Source',
+  side: ['Next Project', 'Previous Project'],
+  icons: ['./assets/icons/Vector.png', './assets/icons/Union.png'],
+  btnForPopup: ['See Live', 'See Source'],
+  nextPrevious: ['./assets/icons/pre.png', './assets/icons/next.png'],
+},
+];
+
+const projectList = document.querySelector('#portfolio ul.grid');
+
+function loadProjects() {
+  for (let i = 0; i < projects.length; i += 1) {
+    const projectItemtemplate = `
+    <li class="project-item ${projects[i].class}">
+      <img
+        class="project-img"
+        src="${projects[i].image}"
+        alt="laptop"
+      />
+      <div class="container container-information hover">
+        <h2 class="heading">${projects[i].title}</h2>
+        <ul class="features">
+          <li class="feature">${projects[i].features[0]}</li>
+          <li class="feature">${projects[i].features[1]}</li>
+          <li class="feature">${projects[i].features[2]}</li>
+        </ul>
+        <button data-id="${projects[i].id}" class="c-projects">
+          See this project &rarr;
+        </button>
+      </div>
+    </li>
+  `;
+
+    projectList.innerHTML += projectItemtemplate;
+  }
+}
+
+window.addEventListener('load', loadProjects());
+
+function createModal(id) {
+  const pCardTemplate = `
+  <div class="overlay hidden">
+      <article class="modal container">
+        <div class="modal__header">
+          <div class="modal__header_title flex space-between">
+            <h1 class="top_title">${projects[id].title}</h1>
+            <button type="button" class="top_btn">
+              &#x2716
+            </button>
```

Comment:
- [ ] Please, relocate the position of the X button in the mobile version 🦅, it should be in the right-top corner similar to the Figma template. 🏒
- [ ] Please, it could be better if you change the size of the fonts in the title 💯.

Your design:

![image](https://user-images.githubusercontent.com/91301423/202871722-4cbfb970-acb0-49dd-8b8c-8f8149ededb3.png)

Figma design:

![image](https://user-images.githubusercontent.com/91301423/202871745-e73265d4-1bd1-4af5-97cc-abc5f6a2c670.png)

Reply:
Hello good evening. I got a review from you now on my you'll request. The other reviewers never told me of this I was just to configure my linters right. Pls I will need your help to approve my project due to limited time. And I will ensure to work on your reviews after that. I will be so grateful :pray:  @Fernando Herrera

Reply:
Hello good evening. I got a review from you now on my you'll request. The other reviewers never told me of this I was just to configure my linters right. Pls I will need your help to approve my project due to limited time. And I will ensure to work on your reviews after that. I will be so grateful :pray: @fherrerao

Reply:
hello @fherrerao 
I have worked on the reviews 
Pls how can you help me so i can submit my project
please @fherrerao